applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -252,6 +242,14 @@ static void stream_qos_set_cb(struct bt_audio_stream *stream)
 		if (ret) {
 			LOG_ERR("Unable to enable stream: %d", ret);
 		}
+
+		if (audio_streams[AUDIO_CH_L].ep->status.state != BT_AUDIO_EP_STATE_STREAMING &&
+		    audio_streams[AUDIO_CH_R].ep->status.state != BT_AUDIO_EP_STATE_STREAMING) {
+			ret = ble_mcs_play_pause(NULL);
```

Comment:
Should add a comment here explaining that the media player is paused after initialization

Reply:
Moved this to stream_enabled_cb()